Library/Homebrew/utils.rb

```diff
@@ -10,8 +10,10 @@
 require "utils/inreplace"
 require "utils/link"
 require "utils/popen"
+require "utils/repology"
 require "utils/svn"
 require "utils/tty"
+require "utils/repology"
```

Comment:
```suggestion
```

Reply:
@MikeMcQuaid are you suggesting we remove this from `utils.rb` to `bump.rb`?

Reply:
No, sorry, just that it's duplicated 😅

Reply:
Okay, thanks for pointing that out.